runs/updateConfig.sh

```diff
@@ -2343,6 +2343,11 @@ updateConfig(){
 	if grep -Fq "socmodul1=soc_idlp2" $ConfigFile; then
 		sed -i "s/^socmodul1=soc_idlp2/socmodul=soc_vwidlp2/g" $ConfigFile
 	fi
-
+	if ! grep -Fq "ppbuchse=" $ConfigFile; then
+		if [[ -f /home/pi/ppbuchse ]]; then
+			ppbuchse=$(</home/pi/ppbuchse)
+			echo "ppbuchse=${ppbuchse}" >> $ConfigFile
+		fi
+	fi
```

Comment:
Mit dieser Variante wird der Eintrag auch aktualisiert, falls er abweicht:
```suggestion
	if [[ -f /home/pi/ppbuchse ]]; then
		ppbuchse=$(</home/pi/ppbuchse)
		if ! grep -Fq "ppbuchse=" $ConfigFile; then
			echo "ppbuchse=$ppbuchse" >> $ConfigFile
		else
			ppbuchseOld=$(grep -F "ppbuchse=" $ConfigFile)
			ppbuchseOld=${ppbuchseOld#ppbuchse=}
			if ((ppbuchseOld != ppbuchse)); then
				sed -i "s/^ppbuchse=.*$/ppbuchse=$ppbuchse/g" $ConfigFile
			fi
		fi
	fi
```